packages/dai-plugin-mcd/contracts/addresses/rinkeby.json

```diff
@@ -0,0 +1,48 @@
+{
+  "DEPLOYER": "0x9EE5e175D09895b8E1E28c22b961345e1dF4B5aE",
+  "MULTICALL": "0x9994692eE12D9Cf8d79065ddD59C32Cd1a5E7ef4",
+  "FAUCET": "0xaa0bDE9f92E0E79ec5597A312802C78f765f8296",
+  "MCD_DEPLOY": "0xEf41F07D6Ae23D5AE9659D7D657bAd7765e5B0bD",
+  "MCD_GOV": "0x81d54B57C604aC34f41458a87DC7708e5eF85D21",
+  "GOV_GUARD": "0xc33D716d576D82e3E9eC94f2580480F2DDb4a00e",
+  "MCD_IOU": "0x60c5DF431c77A8b1f143dB5A9416c9Ce246E8470",
+  "MCD_ADM": "0x3B43dC7090E4be09Cd7fC038f6f0763D78de2e77",
+  "VOTE_PROXY_FACTORY": "0x6241327010C2EF9F97F43b2F896B1E4D5614CB42",
+  "MCD_VAT": "0xca5d845AC105273836e22CdDF71bbf9810489e49",
+  "MCD_JUG": "0xC95280b7077c76A07ABDA8F2e840f777743Bd50c",
+  "MCD_CAT": "0x610b5Bc0b59f6c855F37F7724fDff2205faD2Bfa",
+  "MCD_VOW": "0xD1E52A03BFbaBF782efb65995F854EE8B1d8eeD5",
+  "MCD_JOIN_DAI": "0xA3b0621DA0a55782a86a29E000165de634b259D8",
+  "MCD_FLAP": "0x6Ba68ad57415263dAE9b469918BCB1084E1af92C",
+  "MCD_FLOP": "0x54cccA41b12F5146e7A93Fdc9efB1181E7c14f2D",
+  "MCD_PAUSE": "0xbC20202616F1d546805DE7E35294DC4C63cBdFbA",
+  "MCD_PAUSE_PROXY": "0x736ee81c2B58A153e71Ed55e0675916b61cF8AC7",
+  "MCD_GOV_ACTIONS": "0x936Df20e494fe5f280f07c240FC24b15643D5FbB",
+  "MCD_DAI": "0xb900bc4391013d9388200ED3287B43Ebf068BbDD",
+  "MCD_SPOT": "0x4bf675ed2e74Ae335fBF1d02e4d9d314c24197A7",
+  "MCD_POT": "0xC5B62aFA4135bf269e9CcD1CEfBb8940CE365e79",
+  "MCD_END": "0x6A606a75bA2c27435BF271Eec18d77209A840d29",
+  "MCD_ESM": "0x1310bABCe222964f995fEbE6fe5c5264B6B6275d",
+  "PROXY_ACTIONS": "0x4B59A777E371f7be45aeE99ECBd5Da57B292fc02",
+  "PROXY_ACTIONS_END": "0x13349c9d81BC2f68d86E6D878A848611275eDdE6",
+  "PROXY_ACTIONS_DSR": "0x15652e64b210E4b21804aD42D8B2619692117A26",
+  "CDP_MANAGER": "0xA486787e5E27B7c4A30F7C71591901278Fd9cE81",
+  "GET_CDPS": "0xd68942b52f07756cA2c1F9887750C9353755DC72",
+  "PROXY_FACTORY": "0x99A14C741EE1b912Badb2Cfd2111216B6117115b",
+  "PROXY_REGISTRY": "0xd9CCe40eFb57CBFe222d9505b1746ac3F6fE12a1",
+  "ETH": "0x4B52442F42A158Da7B8217744599Cdb69A8F45AD",
+  "VAL_ETH": "0x7E36C5E46c8f57dE50473f3bBD6D638F3CCA8ba8",
+  "PIP_ETH": "0x7E36C5E46c8f57dE50473f3bBD6D638F3CCA8ba8",
+  "MCD_JOIN_ETH_A": "0x281b88B7B966B091844d005286c3BA0f71a8c16B",
+  "MCD_FLIP_ETH_A": "0x81353cfdd50B0fb139bd43Ebada66a14eCf2169e",
+  "BAT": "0x69283DD6944047b78d94B0E2A64CBc1c32ee6037",
+  "VAL_BAT": "0xccdC6e1F64Bd46Ca80f42A1e27f85b250a40eDeA",
+  "PIP_BAT": "0xccdC6e1F64Bd46Ca80f42A1e27f85b250a40eDeA",
+  "MCD_JOIN_BAT_A": "0xEE74DE857f26476c470142Ae6D0FC4Da62E3f66F",
+  "MCD_FLIP_BAT_A": "0x2fDaed59E1E7763176344466Df5113B91B0A46b6",
+  "PROXY_PAUSE_ACTIONS": "0x81b778996B22004F7ec74Dac59FB1B970aD414e1",
+  "PROXY_DEPLOYER": "0x6B2001c38B2469707eeb07f62B0EE7Cb68c82C3e",
+  "MIGRATION": "0x00000000000000000000000000000000000000000",
+  "MCD_JOIN_SAI": "0x0000000000000000000000000000000000000000",
+  "SAI": "0x4B52442F42A158Da7B8217744599Cdb69A8F45AD"
```

Comment:
The migration and SAI adapter are only used for lookup in the event history functionality and as there is no migration history, the zero addresses are all that is sufficient. 

The SAI token itself does not exist on rinkeby so I reverted to using the WETH address instead.